autogen/structs.rs

```diff
@@ -16,21 +16,20 @@
 
 use serde::de;
 use serde_derive::*;
-use std::{fmt, result, str};
+use std::{convert::TryInto, fmt, result, str};
 
 #[derive(Debug, Deserialize)]
 pub struct Operand {
     pub kind: String,
     #[serde(default)]
     pub name: String,
     #[serde(default)]
-    pub quantifier: String,
+    pub quantifier: Quantifier,
 }
 
 #[derive(Debug, Deserialize)]
 pub struct Instruction {
-    #[serde(default)]
-    pub class: String,
+    pub class: Option<Class>,
```

Comment:
I'm not sure whether an option is best here or whether it would be better to just add a `Normal` class.

Reply:
Given this is just autogen internal, both are fine for me. :)